thirdparty/ofqf/qoscserver.cpp

```diff
@@ -82,8 +82,8 @@ void QOscServer::readyRead()
         		      for ( ; i<size && data[ i ] != char( 0 ); ++i )
         		            path += data[ i ];
         
-        			    while ( data[ i ] != ',' ) ++i;
-        			    ++i;
+        			  while ( data[ i ] != ',' ) ++i;
+        			  ++i;
         		      while ( data[ i ] != char( 0 ) )
```

Comment:
I didn't replace by 4. Have a closer look. There are also Tabs involved in this file. The commit fixes the warning, indeed.